docs_theme/src/scss/styles.scss

```
@@ -0,0 +1,211 @@
+@import "../../node_modules/bootstrap/scss/nav";
+@import "../../node_modules/bootstrap/scss/navbar";
+@import "../../node_modules/bootstrap/scss/card";
+@import "../../node_modules/bootstrap/scss/accordion";
+@import "../../node_modules/bootstrap/scss/breadcrumb";
+@import "../../node_modules/bootstrap/scss/pagination";
+@import "../../node_modules/bootstrap/scss/badge";
+@import "../../node_modules/bootstrap/scss/alert";
+@import "../../node_modules/bootstrap/scss/progress";
+@import "../../node_modules/bootstrap/scss/list-group";
+@import "../../node_modules/bootstrap/scss/close";
+@import "../../node_modules/bootstrap/scss/toasts";
+@import "../../node_modules/bootstrap/scss/modal";
+@import "../../node_modules/bootstrap/scss/tooltip";
+@import "../../node_modules/bootstrap/scss/popover";
+@import "../../node_modules/bootstrap/scss/carousel";
+@import "../../node_modules/bootstrap/scss/spinners";
+@import "../../node_modules/bootstrap/scss/offcanvas";
+@import "../../node_modules/bootstrap/scss/placeholders";
+
+// Helpers
+@import "../../node_modules/bootstrap/scss/helpers";
+
+// Utilities
+@import "../../node_modules/bootstrap/scss/utilities/api";
+
+// Custom Classes
+a {
+  text-decoration: none;
+
+  &:hover,
+  &:active {
+    text-decoration: underline;
+  }
+}
+
+pre.prettyprint {
+  margin-bottom: 20px;
+}
+
+code {
+  @extend .rounded;
+}
+
+pre {
+  @extend .bg-body-tertiary;
+  @extend .rounded;
+}
+
+pre code {
+  border: none;
+}
+
+img {
+  @extend .img-fluid;
+}
+
+.nav-link {
+  padding: 2px 15px 3px;
+}
+
+.body-content {
+  padding-top: 70px;
+  padding-bottom: 70px;
+}
+
+.badges {
+  display: flex;
+  align-items: center;
+  justify-content: end;
+  gap: 8px;
+}
+
+.toclink {
+  color: #333;
+}
+
+// Prettify Code
+.com {
+  color: #93a1a1;
+}
+.lit {
+  color: #195f91;
+}
+.pun,
+.opn,
+.clo {
+  color: #93a1a1;
+}
+.fun {
+  color: #dc322f;
+}
+.str,
+.atv {
+  color: #d14;
+}
+.kwd,
+.prettyprint .tag {
+  color: #1e347b;
+}
+.typ,
+.atn,
+.dec,
+.var {
+  color: teal;
+}
+.pln {
+  color: #48484c;
+}
+
+.prettyprint {
+  padding: 20px;
+}
+.prettyprint.linenums {
+  box-shadow: inset 40px 0 0 #fbfbfc, inset 41px 0 0 #ececf0;
+}
+[data-bs-theme="dark"] {
+  .com {
+    color: #9ea0a0;
+  }
+
+  .lit {
+    color: #4d90fe;
+  }
+
+  .pun,
+  .opn,
+  .clo {
+    color: #9ea0a0;
+  }
+
+  .fun {
+    color: #fe6160;
+  }
+
+  .str,
+  .atv {
+    color: #fea832;
+  }
+
+  .kwd,
+  .prettyprint .tag {
+    color: #4d69a8;
+  }
+
+  .typ,
+  .atn,
+  .dec,
+  .var {
+    color: #38a1a1;
+  }
+
+  .pln {
+    color: #c8c8cc;
+  }
+}
+
+/* Specify class=linenums on a pre to get line numbering */
+ol.linenums {
+  margin: 0 0 0 33px; /* IE indents via margin-left */
+}
+ol.linenums li {
+  padding-left: 12px;
+  color: #bebec5;
+  line-height: 20px;
+  text-shadow: 0 1px 0 #fff;
+}
+
+#table-of-contents {
+  @extend .rounded;
+  padding: 0.5rem;
+  margin-bottom: 1rem;
+}
+
+// Temproary fix for mkdocs settings
```

Fix a typo discovered by `codespell` in pre-commit.
```suggestion
// Temporary fix for mkdocs settings
```

---

To fix the other `codespell` errors, edit the last line of `.pre-commit-config.yaml` to change:
```
    exclude: locale|kickstarter-announcement.md|coreapi-0.1.1.js
# to
    exclude: locale|kickstarter-announcement.md|coreapi-0.1.1.js|docs_theme/package-lock.json
```

---

Thanks, I fixed the problems. I would love to hear your opinions on the progress so far and suggestions for improvement.